packages/notifier/README.md

```diff
@@ -1,59 +1,127 @@
-# NotifierKits and SubscriptionKits
+# PublishKit and Related Types
 
-This package provides two similar abstractions for producing and consuming asynchronous value 
-sequences, the *NotifierKit* and the *SubscriptionKit*. Both let a service notify clients of state changes.
+This package provides an abstraction for production and consumption of
+asynchronous value sequences, the *PublishKit*, along with similar but
+deprecated types *NotifierKit* and *SubscriptionKit*.
```

Comment:
Is `NotifierKit` deprecated? It's not marked so if this PR also making that change it should include the `@deprecated` tag.

Reply:
Yes, AFAIK (search Slack for "deprecated notifier"). Where would you expect to see `@deprecated`?